### ISSUE TITLE
Add basic OCaml bindings

### ISSUE THREAD

**Issue by Spriithy** [reasonml-editor/reasonml-idea-plugin#14]
Since your plugin is the only actual Ocaml plugin up to date on the Jetbrain repositories, could you simply add an extra binding for `.ml` files as OCaml files (with a dedicated icon).

Binding `.ml` files to Reason files works fine, but is not quite the expected behavior.

Thanks in advance of your consideration.

**Comment by giraud** [comment]
I am not sure to understand what you expect:
1. add a file type association to .ml files with a specific icon but still use reason syntax highlighting,
2. change the syntax highlighting to work with ocaml

**Comment by Spriithy** [comment]
I think I would like to have a regular OCaml support since it doesn't appear to be any easy to install & go tool.

That essentially means *change the syntax highlighting to work with ocaml* and *add a file type association to `.ml` files with a specific icon*

**Comment by giraud** [comment]
This is clearly not the goal of this project.
But I understand that seeing some ocaml in parallel with reason might be interesting. 
If I can share enough code between reason and ocaml, I might try to have a very lightweight syntax highlighting, but I think it will never be satisfying for ocaml coders. I will think about it for the next releases, but, really, ocaml is not a priority for this plugin.

**Comment by Spriithy** [comment]
I hope to see you do well in that regard :) !

Good luck & keep up the good work

**Comment by giraud** [comment]
In fact, in version 0.24, I quickly added a lightweight ocaml mode so you can have file association and no incorrect parser errors.
It's a start. but I might be not very active on the ocaml side.

![issue_14](https://user-images.githubusercontent.com/239612/32772758-27b969ae-c927-11e7-9e30-cf139698f202.png)

I hope it's a little better like that.

**Comment by Spriithy** [comment]
Looks great !

I think it is now suited to address this topic in #13
